sortbootorder.c

```diff
@@ -493,6 +497,7 @@ static void show_boot_device_list(char buffer[MAX_DEVICES][MAX_LENGTH],
 #endif
 	printf("  w Enable BIOS write protect - Currently %s\n",
 		(spi_wp_toggle) ? "Enabled" : "Disabled");
+	printf("  z Clock menu\n")
```

Comment:
```suggestion
	printf("  z Clock menu\n");
```

Was it even build tested?